### PR TITLE
SCRUM-1185: Add "Note" object.

### PR DESCRIPTION
**Pull request by gildossantos**
NB - I'm leaving EntityStatement in place now so that I can coordinate the replacement of it with the expression working group, but I wanted to get the new Note in place ASAP.

### CODE REVIEW

File: model/schema/core.yaml [hunk 1]
```diff
@@ -293,6 +309,32 @@ slots:
...
 
+  free_text:
+    description: >-
+      A free text string that describes some aspect of an entity.
+    range: string
+
+  note_type:
+    description: >-
+      The type of note: e.g., cytology, comment, summary.
+    range: OntologyTerm
```
Comment (chris-grove) [inline model/schema/core.yaml:320]: The range of 'note_type' should be 'VocabularyTerm' from the controlledVocabulary.yaml, not 'OntologyTerm'; this also means that we'll need to add to the 'imports' list 'controlledVocabulary'

File: model/schema/core.yaml [hunk 2]
```diff
@@ -144,6 +144,22 @@ classes:
       - generated_by
       - manufactured_by
 
+  Note:
+    description: >-
+      Free-text describing some attribute of an entity.
```
Comment (chris-grove) [inline model/schema/core.yaml:149]: I would prefer a description like: "Note object for capturing free-text describing some attribute of an entity, coupled with a 'note type', private boolean, and an optional list of references." Basically, I want to make sure that we don't just define the Note object as only free-text.

**Review by chris-grove — CHANGES_REQUESTED** [review]
There are two comments to address plus the addition of 'controlledVocabulary' to the list of core.yaml imports.

File: model/schema/core.yaml (hunk 2 above)
Comment (gildossantos) [reply model/schema/core.yaml]: Yes, that makes sense. Done.

File: model/schema/core.yaml (hunk 1 above)
Comment (gildossantos) [reply model/schema/core.yaml]: Sorry. Fixed now.

**Review by chris-grove — APPROVED** [review]
Looks good, thanks!

**Pull request merged**